probe-rs/src/architecture/arm/sequences/atsame5x.rs

```diff
@@ -0,0 +1,379 @@
+//! Sequences for ATSAM D5x/E5x target families
+
+use super::ArmDebugSequence;
+use crate::{architecture, DebugProbeError, Error, Memory, Permissions};
+use bitfield::bitfield;
+use std::sync::Arc;
+
+use anyhow::Result;
+
+bitfield! {
+    /// Device Service Unit Control Register, DSU - CTRL
+    #[derive(Copy, Clone)]
+    struct DsuCtrl(u8);
+    impl Debug;
+    // Chip-Erase
```

Comment:
Can you make all of those (the ones further down as well) a doc comment? :)

Reply:
Sure, will fix